README.md

```diff
@@ -6,4 +6,39 @@
 [![Build Status](https://travis-ci.org/fluture-js/concurrify.svg?branch=master)](https://travis-ci.org/fluture-js/concurrify)
 [![Code Coverage](https://codecov.io/gh/fluture-js/concurrify/branch/master/graph/badge.svg)](https://codecov.io/gh/fluture-js/concurrify)
 
-Does nothing yet.
+Turn non-concurrent [FantasyLand 3][FL3] Applicatives concurrent.
+
+## Usage
+
+```js
+//The concurrify takes four arguments, explained below.
+const concurrify = require('concurrify');
+
+//We load the Type Representative of the Applicative we want to transform.
+const Future = require('fluture');
+
+//We create a "zero" instance and a "race" function for "Alternative".
+const zero = Future(() => {});
+const alt = Future.race;
+
+//We create an override "ap" function that works in parallel.
+const ap = (mx, mf) => Future.both(mx, mf).map(([x, f]) => f(x));
+
+//Calling concurrify with these arguments gives is a new Type Representative.
+const ConcurrentFuture = concurrify(Future, zero, alt, ap);
+
+//We can use our type as such:
+ConcurrentFuture(Future.of(1)).sequential.fork(console.error, console.log);
+```
+
+## API
+
+```hs
+concurrify :: (Applicative m, Alternative n)
+           => (TypeRep m, m a, (m a, m a) -> m a, (m a, m (a -> b)) -> m b)
+           -> n m
```

Comment:
What does this mean?

```hs
data Concurrent a b = Applicative a => Concurrent {sequential :: a b}
```
```txt
<interactive>:8:23:
    Data constructor ‘Concurrent’ has existential type variables, a context, or a specialised result type
      Concurrent :: forall (a :: * -> *) b.
                    Applicative a =>
                    a b -> Concurrent a b
      (Use ExistentialQuantification or GADTs to allow this)
    In the definition of data constructor ‘Concurrent’
    In the data declaration for ‘Concurrent’
```

Is it something like: "Hey, you're doing something quite *meta* here. That's only supported with a plugin"?